sympy/vector/functions.py

```diff
@@ -28,7 +28,8 @@ def extended_express(expr,coordsys):
     >>> from sympy.vector import extended_express, CoordSys3D
     >>> from sympy import pi
     >>> N=CoordSys3D('N')
-    >>> C=N.create_new('C',transformation='cylindrical',vector_names=('r','theta','z'))
+    >>> C=N.create_new('C',transformation='cylindrical', \
```

Comment:
The tailing backslash is not needed inside parentheses. Space on either side of `=` and after first comma.

Reply:
sure

Reply:
actually this has broken the travis build under doctests, so reintroducing - gives unexpected 'EOF' on travis but not locally